### DISCUSSION TITLE
Github is a collection of repositories: So where are my repositories?

### DISCUSSION THREAD

**Discussion by loteque** [community/community#127510] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

When I point my address bar to github.com I want my cursor in a search bar for all my repositories and the repositories to which I am contributing as well as my forks of projects to which I am not yet contributing.
I want to see a list of these things and to be able to \<Tab\> through them.
Much appreciated,
loteque

**Comment by loteque** [comment, marked as answer]
**`shift`+`k` for those that have the same issue.**
_`ctrl`+`shift`+`k` for the github.com `command palette`_
`ctrl`+`shift`+`k`, `backspc` will get you to search as well.

It would be nice to have a less busy account homepage with our repos and a search front and center, but I think we area fairly committed to turning github into linkedin.com these days.

**Reply by ebndev** [reply]
Hi @loteque, thanks for the feedback and for posting your solution.

In case this might also be helpful, you can see a full list of your repositories by clicking on your profile picture in the top right and bookmark "Your Repositories".

**Reply by loteque** [reply]
I don't want to click. I want to type in github.com and be able to tab with my keyboard through a sortable list of repositories.

When the page is loaded I want my full list of repos loaded (or loading, background on_scroll) And I want my cursor focused on the first in the list.

Don't get me sideways... I love the social features of github.com. but those features are centered around effort and all that effort is organized around repositories.

I just think we have room to get back to some first principles design here. 

Let's have social features that respect our effort.

**Comment by vishal11u** [comment]
response #1
